src/DIRAC/Resources/Computing/SingularityComputingElement.py

```diff
@@ -318,7 +285,8 @@ def __createWorkArea(self, jobDesc=None, log=None, logLevel="INFO", proxy=None):
             # Extra Wrapper (Container DIRAC installer)
             wrapSubs = {
                 "next_wrapper": wrapperPath,
-                "install_args": self.__getInstallFlags(infoDict),
+                "dirac_project": self.__getInstallFlags(infoDict)[0],
+                "version": self.__getInstallFlags(infoDict)[1],
```

Comment:
I'd be tempted to to set `installFlags = self.__getInstallFlags(infoDict)` and then use installFlags[0] and installFlags[1] here to avoid running the function twice.